internal/controller/prefix_controller.go

```diff
@@ -132,9 +132,17 @@ func (r *PrefixReconciler) Reconcile(ctx context.Context, req ctrl.Request) (ctr
 			return ctrl.Result{}, err
 		}
 
+		if prefixClaim.Status.ParentPrefix == "" {
+			// the parent prefix is not computed
+			logger.Info("the parent prefix is not computed")
```

Comment:
you could set the ready condition of the prefix to false here.

Reply:
It's missed, thanks for flagging this!

Reply:
Fixed